src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -129,7 +134,8 @@ object SamResourceTypes {
 @Lenses final case class CreateResourceRequest(
     resourceId: ResourceId,
     policies: Map[AccessPolicyName, AccessPolicyMembership],
-    authDomain: Set[WorkbenchGroupName])
+    authDomain: Set[WorkbenchGroupName],
+    returnResource: Boolean = false)
```

Comment:
i saw a little bit of discussion on slack about this-- so akka-http doesn't support query params on a post?

just asking because this feels kind of weird but if it's necessary then i'm okay with it

Reply:
also should this be an Option[Boolean]?  the automated tests seem unhappy with this.

```
org.broadinstitute.dsde.workbench.service.RestException: {
  "statusCode": 400,
  "source": "rawls",
  "timestamp": 1570641340939,
  "causes": [],
  "stackTrace": [],
  "message": "Sam call to HttpMethod(POST) /api/resources/v1/workspace failed with error HttpEntity.Strict(text/plain; charset=UTF-8,The request content was malformed:\nObject is missing required member 'returnResource')"
}
```

Reply:
yeah -- I couldn't get it to work and banged on it for like an hour.  Basically it seems to support getting the parameter, but not it's value (it was always empty string). 

Moving to option though

Reply:
ahhh sorry - ignore my other comments about `Option`, I see now why you made that change.

Reply:
👍